### ISSUE TITLE
icp cannot provide source resource or replica number

### ISSUE THREAD

**Issue by alanking** [irods/irods#6498]
- [ ] main
- [ ] 4-3-stable
---
## Feature

`icp` has the ability to specify a destination resource via the `-R` option. It does not have an option to specify a source resource or replica number for the source data object. For many iCommands, this is `-S` for source resource and `-n` for replica number.

As with other iCommands, `-S` and `-n` should be considered incompatible parameters.

The recursive flag `-r` should also be considered while implementing this feature: What happens if the specified source resource or replica number is not valid for every data object in a collection?

**Comment by trel** [comment]
I think your last point is the same as https://github.com/irods/irods/issues/6444.

**Comment by alanking** [comment]
Right. A possible resolution for this issue is that this should not be allowed. Haven't really thought through/discussed it all the way yet
